src/main/java/it/pagopa/pm/gateway/utils/VPosUtils.java

```diff
@@ -0,0 +1,63 @@
+package it.pagopa.pm.gateway.utils;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static it.pagopa.pm.gateway.constant.VposConstant.ID_PSP_POSITION;
+
+@Slf4j
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Component
+public class VPosUtils {
+
+    private static final String ASTERISK_SPLIT_CHAR = "\\*";
+    private static final String PIPE_SPLIT_CHAR = "\\|";
+    private final Map<String, List<String>> vposShopMap = new HashMap<>();
+
+    @Autowired
+    private Environment environment;
```

Comment:
Is there a specific reason why you switched from `@Value` injection to autowiring the environment?

Reply:
For junits, since the property is used in the postConstruct with the @value annotation it could not retrieve the value of the variable

Reply:
Ok I'll do it in another PR, so we can bring everything into a test environment and we can test it